R/query.R

```diff
@@ -43,15 +43,23 @@
 Query <- function(queryString) {
   # Encode the query to REST URL
   urlEncodedQuery <- URLencode(queryString, reserved = TRUE)
-  reqUrl <- paste0("http://api.datacommons.org/query?sparql=", URLencode(urlEncodedQuery), "&key=", Sys.getenv("API_KEY"))
+  reqUrl <- paste0("http://api.datacommons.org/query?sparql=",
```

Comment:
I'm curious why this isn't a wrapper of the python library as well?

Reply:
The python client doesn't have a Query function (it has an _execute method on the query class)

Reply:
ah, got it.

@ACscooter, does the python library need query?

Reply:
The Python Client Library doesn't need a query object per se. We could implement it as a function that accepts a query string.

This way the R package could simply wrap that function?